app/lib/helpers/getDurationFromDates.ts

```diff
@@ -0,0 +1,27 @@
+import { DateTime } from "luxon";
+
+const getDurationFromDates = (startDate: string, endDate: string) => {
```

Comment:
Can we have a small comment indicating the format this function returns the duration in?